src/goGraph/storybook/StoryWrapper.tsx

```diff
@@ -8,13 +8,24 @@ import { useRdfActionReducer } from '../../core/state/useRdfState';
 import { RdfGoGraph } from '../RdfGoGraph';
 import { getDefaultLayoutConfig, GoGraphLayout } from '../layout';
 import { GoGraphOptions } from '../types/component.types';
+import { getNodeSymbolTemplate, NodeSymbol } from '../../symbol-api';
+import { UiNodeSymbol } from '../../core/ui/applyPatch';
+import { NodeSymbolToUiNodeSymbol } from '../../core/ui/defaultSymbolProvider';
 
 export type SparqlWrapperProps = {
 	turtleString: string;
 	layout: GoGraphLayout;
 	selectionEffect?: SelectionCallback;
 };
 
+function symbolProvider(id: string, rotation?: number): UiNodeSymbol | undefined {
```

Comment:
Remove rotation param or prefix it with `_` to remove warning

Reply:
Fixed